spec/knapsack_pro/config/temp_files_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe KnapsackPro::Config::TempFiles, :clear_tmp do
+  describe '.temp_directory_path' do
+    let(:gitignore_file_path) { '.knapsack_pro/.gitignore' }
+
+    subject { described_class.temp_directory_path }
+
+    it 'returns temporary directory path' do
+      expect(subject).to eq '.knapsack_pro'
+    end
+
+    it 'creates .gitignore file' do
+      expect(File.exists?(gitignore_file_path)).to be false
```

Comment:
FYI there is a nice DSL for expectations like that https://relishapp.com/rspec/rspec-expectations/v/2-0/docs/matchers/expect-change

e.g. `expect{Counter.increment}.to change{Counter.count}.from(0).to(1)`

(just wanted to share this in case you forgot about it, the current one is cool by all means)